internal/cmd/commands/authmethodscmd/oidc_funcs.go

```diff
@@ -42,6 +43,7 @@ const (
 	caCertFlagName                            = "idp-ca-cert"
 	allowedAudienceFlagName                   = "allowed-audience"
 	claimsScopes                              = "claims-scopes"
+	accountClaimMaps                          = "account-claim-maps"
```

Comment:
Honest question since I'm not as familiar with oidc as you,  Is the `account-` prefix needed and useful here?

Reply:
not really but I'm trying to be clear that it's the claim maps that will be used for the account.